### PR TITLE
Jmx history v3

### PR DESCRIPTION
**Pull request by pnowojski**
Supersedes https://github.com/prestodb/presto/pull/5045

### CODE REVIEW

**Comment by pnowojski** [comment]
@martint please take a look on this and on my responses to your comments in previous version

**Comment by pnowojski** [comment]
@martint  again, applied changes, please take a look on them here

File: presto-docs/src/main/sphinx/connector/jmx.rst [hunk 1]
```diff
@@ -18,21 +21,35 @@ To configure the JMX connector, create a catalog properties file
...
+To enable periodical dumps, define following properties:
+
+.. code-block:: none
+
+    jmx.dump-tables=java.lang:type=Runtime,com.facebook.presto.execution.scheduler:name=NodeScheduler
+    jmx.dump-period=10s
+    jmx.max-entries=86400
+
+``dump-tables`` is a comma separated list of Managed Beans (MBean). It specifies
+which MBeans will be sampled and stored in memory every ``dump-period`` milliseconds.
```
Comment (martint) [inline presto-docs/src/main/sphinx/connector/jmx.rst]: dump-period takes an interval with arbitrary unit, so "milliseconds" is incorrect.

File: presto-docs/src/main/sphinx/connector/jmx.rst [hunk 2]
```diff
@@ -45,11 +62,27 @@ The following query shows the open and maximum file descriptor counts
...
 
 .. code-block:: none
 
      openfiledescriptorcount | maxfiledescriptorcount
     -------------------------+------------------------
                          329 |                  10240
     (1 row)
+
+The second schema is called ``history`` and contains dumped tables as configured
+in ``jmx.properties``. It contains ``dump-tables`` with exactly the same
```
Comment (martint) [inline presto-docs/src/main/sphinx/connector/jmx.rst]: Technically, the properties file could be named anything. I'd say something like:

The ``history`` schema contains the list of tables configured in the connector properties file.
The tables have the same columns as those in the ``current`` schema, but with an additional 
timestamp column ...

**Comment by pnowojski** [comment]
@martint applied doc changes

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**